apps/edxapp/templates/services/nginx/configs/lms.conf.j2

```diff
@@ -44,6 +44,20 @@ server {
     try_files $uri @proxy_to_lms_app;
   }
 
+{% if edxapp_nginx_admin_ip_withelist | length > 0 %}
```

Comment:
Should we define two separated lists for the CMS and the LMS ?

Reply:
Yes I will do that